docs/inspector.md

```diff
@@ -0,0 +1,43 @@
+# Hermes Inspector
+
+React Native for Windows with Hermes engine now supports direct JavaScript runtime inspection using tools such as Chrome or Edge devtools, VSCode debugger, Flipper etc. by implementing an in-process Chrome Debug Protocol server.  Please note that it is fundamentally different from "Remote JS Debugging", which loads the JavaScript bundle into a remote Chrome browser session with dupelx communication over IPC channels.
```

Comment:
It would be nice to limit line length to 80 or 120 to make it easier to review and read. The markdown format allows to write the same paragraph on multiple lines.